src/project/qml/MuseScore/Project/internal/devtools/convertfiletoscoredevtoolsmodel.cpp

```
@@ -34,33 +34,10 @@ ConvertFileToScoreDevToolsModel::ConvertFileToScoreDevToolsModel(QObject* parent
-        const IInteractive::ButtonData closeBtn(IInteractive::Button::Close, "Close");
-
-        interactive()->info("Your score is ready!", "", { openScoreBtn, closeBtn }, closeBtn.btn)
-        .onResolve(this, [this, path, openScoreBtn](const IInteractive::Result& res) {
-            if (res.isButton(openScoreBtn.btn)) {
-                dispatcher()->dispatch("file-open", ActionData::make_arg1<QUrl>(path.toQUrl()));
-            }
-        });
-    });
-}
-
 void ConvertFileToScoreDevToolsModel::selectAndConvertFiles()
```

Cppcheck (2.21.0)

[style] The function 'selectAndConvertFiles' is never used.

(unusedFunction)